Library/ViewModels/LiveStreamChatViewModel.swift

```diff
@@ -204,6 +216,12 @@ LiveStreamChatViewModelOutputs {
       return AppEnvironment.current.currentUser != nil
     }
 
+    self.textFieldShouldChangeCharactersInRangeReturnProperty <~
+      textFieldShouldChangeCharactersInRangeProperty.signal.skipNil().map { currentString, range, string in
+        return (currentString as NSString)
+          .replacingCharacters(in: range, with: string).characters.count <= maxMessageLength
```

Comment:
I always thought this delegate method was a bit awkward, and in this case a paste that brings things over the limit won't be applied/truncated. A couple solutions:

1. Allow users to go over the limit (a la Twitter), but turn the counter red and disable submission.
2. Have a VM output tracking what the current chat input is and use `.editingChanged` as an input that can handle the truncation.

The former seems a bit less troublesome to implement.